test/Microsoft.ML.CpuMath.UnitTests.netcoreapp/UnitTests.cs

```diff
@@ -116,7 +116,7 @@ public void MatMulTranTest(int matTest, int srcTest, int dstTest, float[] expect
         [InlineData(0, 0, 0, new float[] { 38.25002f, 38.25002f, 38.25002f, 38.25002f, 38.25002f, 38.25002f, 38.25002f, 38.25002f })]
         [InlineData(1, 1, 0, new float[] { 910f, 2190f, 3470f, 4750f, 6030f, 7310f, 8590f, 9870f })]
         [InlineData(1, 0, 1, new float[] { 95f, 231f, 367f, 503f, 639f, 775f, 911f, 1047f, 1183f, 1319f, 1455f, 1591f, 1727f, 1863f, 1999f, 2135f })]
-        public void MatMulPATest(int matTest, int srcTest, int dstTest, float[] expected)
+        public void MatMulPTest(int matTest, int srcTest, int dstTest, float[] expected)
```

Comment:
Why isn't this test called `MatTimesSrcTest` to match the function it is testing?

Reply:
A couple of overload are present for ```MatTimesSrc``` . And these overloads also contains flags such as ```trans``` in order to decide what version of matrix multiplication will finally be executed. 
So we are using this name for test to show which matrix multiplication is being used

Reply:
I think this test should be named `MatTimesSrcSparse`. It would match the name of the method it is testing, and gives the indication for which overload it is representing (the sparse vector overload).